features/account_settings.feature

```diff
@@ -0,0 +1,8 @@
+
+Feature: Checking account settings
+
+Scenario: Check account settings
+    When we are on Account settings Page
+    When we enter login
```

Comment:
Do we have login form on account settings page?
I guess we get redirected to login page in this flow.
Could adding "background" steps for login help clarify the scenario? https://pythonhosted.org/behave/gherkin.html#background

Reply:
I thought about it too, but this is only way (for me now):
We cant first make login and go to settings page, bcz we will redirect again.
So my solution is give as background step to redirect to account settings page, and in main step enter login, pwd and check settings.